Gems/EMotionFX/Code/EMotionFX/Source/Mesh.cpp

```diff
@@ -1418,6 +1460,77 @@ namespace EMotionFX
         return hasIntersected;
     }
 
+    bool Mesh::Intersects(
```

Comment:
I thought EMFX meshes were unified with Atom meshes now? Is there not some heavily optimized routine for this already for Atom meshes?

Reply:
Atom doesn't have a mesh suitable for main memory and CPU based deformations. So we're keeping the EMFX runtime meshes until that is available, as game servers often don't have a GPU but are responsible for hit detection - and the point was made by someone that you might want to have full accuracy on the mesh rather than just hit volumes.

Reply:
I am not sure if this is enough justification to keep and maintain all this code though.

Is anyone from @o3de/sig-graphics-audio who can update us on the path forward for main memory optimized versions of Atom meshes?

Reply:
Perhaps @amzn-tommy might know about this in more detail (tagging @galibzon for visibility too)